content/docs/welcome-guide/requirements.md

```diff
@@ -12,20 +12,26 @@ Open 3D Engine (O3DE) has a minimum set of hardware and software requirements fo
 
 Minimum hardware requirements for development include the following:
 
++ Windows 10 64-bit version 1809 or Ubuntu 18.04 LTS
 + Quad-core Intel or AMD processor, 2.5 GHz or faster
 + 8 GB RAM
-+ DirectX 12 or Vulkan-compatible video card with 2 GB VRAM minimum
++ DirectX 12, Vulkan-compatible, or Metal-compatible video card with 2 GB VRAM minimum
   + 4 GB VRAM recommended
   + NVIDIA GeForce GTX 1060 or better, driver version 465.89 or later recommended
   + AMD Radeon Pro 560 or better
   + Intel HD 630 or better
 + 40 - 100 GB of free disk space
 + 1366 x 768 px screen resolution
++ Shader Model 6.2 (or Shader Model 6.3 to use raytracing features)
 
 {{< note >}}
 RAM and free disk space requirements are dependent on the options that you select when configuring your project in O3DE.
 {{< /note >}}
 
+{{< note >}}
+Developing with O3DE on macOS is an experimental feature. At a minimum, you need macOS Catalina and XCode 11.
```

Comment:
I think a note on OS support belongs in the next section on Software prereqs.

Reply:
Metal should also be pulled down into this section, along with the other macOS requirements - M1/ARM arch isn't supported in particular.

Reply:
What is the proper name for M1/ARM arch? I found it referred to as an "Apple R1 processor" that is "ARM-based". So I was planning to just refer to is as "Apple R1 processor" unless there is a more accurate title.

Also, do you suggest I keep this informatin in a note? Currently the docs are Windows (and now Linux) focused. If we were to add Macs to the mix, I think we need to further rearrange the list of requirements so it's clear. 

Since Mac support is still experimental and we don't have futher documentation on it, I lean towards keeping it in a note for now.

Reply:
I agree that mac support should be in the note. As far as chipset support: Probably the easiest way to say it is that "Intel x86_64" macs are supported.